css/properties/appearance.json

```diff
@@ -60,6 +63,9 @@
               }
             ],
             "firefox_android": [
+              {
+                "version_added": "80"
```

Comment:
This looks good, but it looks like some Firefox Android data is missing, if you run `npm run mirror -- --modify=always firefox_android css`.